x-pack/plugins/cloud_integrations/cloud_chat/server/routes/chat.test.ts

```diff
@@ -5,43 +5,42 @@
  * 2.0.
  */
 
+import { securityMock } from '@kbn/security-plugin/server/mocks';
```

Comment:
I still needed to import/keep this mock: `securityMock.createMockAuthenticatedUser` is not available in the mocks from core.

Reply:
This is fine until we move the implementation to core.